### DISCUSSION TITLE
Hyperlink to another slide

### DISCUSSION THREAD

**Discussion by MeganMcAuliffe** [davidgohel/officer#512] [Ideas]
Hi! I just was wondering if this feature was ever implemented?:

https://stackoverflow.com/questions/66886292/using-officer-in-r-to-hyperlink-to-another-slide-within-a-flextable-cell

**Comment by davidgohel** [comment]


**Reply by davidgohel** [reply]
"No, there hasn't been any specific follow-up on this."
